src/main/java/cloud/cholewa/game/logic/GameStarter.java

```diff
@@ -0,0 +1,41 @@
+package cloud.cholewa.game.logic;
+
+import cloud.cholewa.game.Player;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class GameStarter {
+
+    public static int setNumberOfPlayers(Scanner scanner) {
+        int number;
+
+        System.out.print("Podaj liczbę graczy (2-4): ");
+        number = scanner.nextInt();
+
+        if (number < 2 || number > 4) {
+            System.out.println("Błędna ilość graczy !");
```

Comment:
Wcześniej używałeś osobnej klasy "słownikowej" do trzymania tekstów, warto to myślę kontynuować, nawet tylko w przypadku error messages. Albo zrezygnować. Generalnie albo jedno rozwiązane, albo drugie. Najgorsze co może być to część tekstów trzymana w dedykowanym miejscu, a część rozrzucona po systemie.

Reply:
Utworzyłem osobną klasę z tekstami występującymi w grze. I trochę rozbiłem to na większą ilość klas, żeby rozbić odpowiedzialność poszczególnych metod.